rpm/__init__.py

```diff
@@ -79,21 +94,110 @@ def try_path(path: str) -> bool:
     return False
 
 
-def initialize() -> None:
+def import_helper(path: Path, suffixes: List) -> None:
```

Comment:
Function name should be a verb, not a noun:

```suggestion
def reload_module(path: Path, suffixes: List[str]) -> None:
```